ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/transactions/layered/LayeredPendingTransactionsTest.java

```diff
@@ -207,14 +208,14 @@ public void addRemoteTransactions() {
         createRemotePendingTransaction(transaction0), Optional.empty());
     assertThat(pendingTransactions.size()).isEqualTo(1);
 
-    assertThat(getAddedCount(REMOTE, NO_PRIORITY, layers.prioritizedTransactions.name()))
+    assertThat(getAddedCount(REMOTE, NO_PRIORITY, NEW, layers.prioritizedTransactions.name()))
```

Comment:
I noticed that you are only using the reason "NEW" in this file. Is that because other reasons don't make a difference here? Do the tests in LayersTest cover all the functionality that has changed with the "reasons"?

Reply:
Using only `NEW` here because we are only adding new txs to the pool, and not moving existing txs between layers. Will review if more tests are needed to cover everything.

Reply:
added tests to cover also the move of txs between layers